src/legacy/php/actions/get_filtered_list_topics.php

```diff
@@ -390,62 +401,71 @@
         // все хранимые раздачи
         // не выбраны статусы раздач
         if (empty($filter['filter_tracker_status'])) {
-            throw new Exception('Не выбраны статусы раздач для трекера');
+            $logger->error("No statuses to filter", []);
+            return $defaultResponse;
         }
 
         if (empty($filter['keeping_priority'])) {
             if ($forum_id == -5) {
                 $filter['keeping_priority'] = [2];
             } else {
-                throw new Exception('Не выбраны приоритеты раздач для трекера');
+                $logger->error("No priorities to filter", []);
+                return $defaultResponse;
             }
         }
 
         if (empty($filter['filter_client_status'])) {
-            throw new Exception('Не выбраны статусы раздач для торрент-клиента');
+            $logger->error("No statuses to filter in torrent-client", []);
+            return $defaultResponse;
         }
 
         // некорретный ввод значения сидов или количества хранителей
         $filters_hints = [
-            "filter_rule_interval" => "сидов",
-            "keepers_filter_rule_interval" => "количества хранителей",
+            "filter_rule_interval",
+            "keepers_filter_rule_interval",
         ];
-        foreach ($filters_hints as $filter_name => $hint) {
+        foreach ($filters_hints as $filter_name) {
             if (isset($filter['filter_interval']) || $filter_name == "keepers_filter_rule_interval") {
                 if (
                     !is_numeric($filter[$filter_name]['from'])
                     || !is_numeric($filter[$filter_name]['to'])
                 ) {
-                    throw new Exception('В фильтре введено некорректное значение ' . $hint);
```

Comment:
This generates a message for the user to be shown on the web-page and it is useful enough to still have it.

Reply:
Added to checklist.